sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -2,13 +2,7 @@
 
 ## 4.0.0-beta.2 (2022-03-08)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- second preview
```

Comment:
1. Is there really nothing to put here?
2. Start with capital letter and end with period.